### PR TITLE
LPS-69587 Add new unit tests

### PR DESCRIPTION
**Pull request by diogo-lins**

### CODE REVIEW

**Comment by brunobasto** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select.soy [hunk 1]
```diff
@@ -27,45 +28,116 @@
 {template .select autoescape="deprecated-contextual"}
```
Comment (brunobasto) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select.soy]: Please run check_sf on this file before sending. Here's the output: 

dynamic-data-mapping-type-select ➤ check_sf src/main/resources/META-INF/resources/**/*.soy                                                                                                     git:pr-796
File: src/main/resources/META-INF/resources/select.soy
    Line 41: Sort attribute values: input-group-container form-builder-select-field
    Line 55: Sort attribute values: select-field-trigger form-control
    Line 55: Sort attributes: href dir

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select_field.js [hunk 2]
```diff
@@ -191,10 +244,70 @@ AUI.add(
...
+										}
+									);
+								}
+							);
+						}
+
+						return optionsSelected;
+					},
+
+					_isListOpen: function() {
```
Comment (brunobasto) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select_field.js]: Function only returns sometimes?

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/testJS/select_field.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+			}
+		);
+
+		describe(
+			'.clickItem()',
+			function() {
+				it(
+					'should click item and get it value',
+					function() {
+
```
Comment (brunobasto) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/testJS/select_field.js]: Remove whitespace

**Comment by brunobasto** [comment]
Also, we are missing the gradle part that enables the CI to run these tests. Can you add it?

**Comment by diogo-lins** [comment]
I'll add now.

**Comment by brunobasto** [comment]
ci:stop

**Comment by brunobasto** [comment]
ci:retest

**Pull request closed without merging**